skyportal/facility_apis/observation_plan.py

```diff
@@ -397,6 +397,20 @@ def generate_plan(
             'doAvoidGalacticPlane': request.payload.get("galactic_plane", False),
             # galactic latitude to exclude
             'galactic_limit': request.payload.get("galactic_latitude", 10),
+            # threshold number of fields?
+            'doMaxTiles': request.payload.get("max_tiles", False),
+            # Maximum number of fields to consider
+            'max_nb_tiles': [request.payload.get("max_nb_tiles", 100)]
+            * len(request.payload["filters"].split(",")),
+            # balance observations by filter
+            'doBalanceExposure': request.payload.get("balance_exposure", False),
+            # slice observations by right ascension
+            'doRASlice': request.payload.get("ra_slice", False),
+            # right ascension block
+            'raslice': [
```

Comment:
The capitalization on `raslice` is inconsistent with the other parameters.

Reply:
Yeah someday someone will clean up the parameter names